solutions/Factorial/prod15.js

```diff
@@ -0,0 +1,6 @@
+for(var x = 1; x > 16; x++){
+    var total = 1;
+    var newtotal *= total * x;
+}
```

Comment:
You're obviously headed in the right direction, but just a couple things:

- the loop won't even execute once since your condition is `x > 16` when you initialize `x` with `1`. You prolly wanted `x < 16`.
- try to declare your variables outside of the loop or else you'll get them reset upon each iteration.